virttest/env_process.py

```diff
@@ -1285,6 +1287,9 @@ def preprocess(test, params, env):
         # work on connect_uri as default behavior.
         os.environ['LIBVIRT_DEFAULT_URI'] = connect_uri
         if params.get("setup_libvirt_polkit") == "yes":
+            if (libvirtd_ver.version_compare(5, 6, 0) and
+               libvirtd_decorator.get_libvirtd_split_enable_bit()):
```

Comment:
I suggest we move L1290 L1291 together in near feature. But we can do it in another PR as there are many  places like this.